app/assets/stylesheets/components/_footer.scss

```diff
@@ -1,16 +1,19 @@
 
 .footer-container {
 	opacity: 0.2;
-	width: 97%;
+	width: 100%;
 	border-top: 1px solid $dark-gray;
 	display: flex;
 	align-items: center;
 	justify-content: space-between;
 	padding: 1px 0;
```

Comment:
There seems to be a padding, padding-left, and padding right. I think this can be simply written as: padding: 1px 15px 1px 15px;